notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,714 @@
+    "\n",
+    "```\n",
+    "model_repository/\n",
+    "`-- fil\n",
+    "    |-- 1\n",
+    "    |   `-- xgboost.model\n",
+    "    `-- config.pbtxt\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create directory to save the model\n",
+    "![ ! -d \"/data/model_repository\" ] && mkdir -p /data/model_repository/fil/1\n",
+    "\n",
+    "# Save your xgboost model as xgboost.model\n",
+    "model.save_model('/data/model_repository/fil/1/xgboost.model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**2. Create and save config.pbtxt**\n",
+    "\n",
+    "To deploy the model in Triton Inference Server, we need to create and save a protobuf config file called config.pbtxt under `model_repository/fil/` directory that contains information about the model and the deployment. Sample config file is available here: [link](https://github.com/triton-inference-server/fil_backend#configuration)\n",
+    "\n",
+    "Essentially, the following parameters need to be updated as per your configuration\n",
+    "\n",
+    "```\n",
+    "name: \"fil\"                              # Name of the model directory (fil in our case)\n",
+    "backend: \"fil\"                           # Triton FIL backend for deploying forest models\n",
+    "max_batch_size: 8192\n",
+    "input [\n",
+    " {\n",
+    "    name: \"input__0\"\n",
+    "    data_type: TYPE_FP32\n",
+    "    dims: [ 9 ]                          # Input feature dimensions, in our sample case it's 9\n",
+    "  }\n",
+    "]\n",
+    "output [\n",
+    " {\n",
+    "    name: \"output__0\"\n",
+    "    data_type: TYPE_FP32\n",
+    "    dims: [ 2 ]                          # Output 2 for binary classification model\n",
+    "  }\n",
+    "]\n",
+    "instance_group [{ kind: KIND_GPU }]\n",
+    "parameters [\n",
+    "  {\n",
+    "    key: \"model_type\"\n",
+    "    value: { string_value: \"xgboost\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"predict_proba\"\n",
+    "    value: { string_value: \"false\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"output_class\"\n",
+    "    value: { string_value: \"true\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"threshold\"\n",
+    "    value: { string_value: \"0.5\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"algo\"\n",
+    "    value: { string_value: \"ALGO_AUTO\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"storage_type\"\n",
+    "    value: { string_value: \"AUTO\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"blocks_per_sm\"\n",
+    "    value: { string_value: \"0\" }\n",
+    "  }\n",
+    "]\n",
+    "\n",
+    "dynamic_batching {\n",
+    "  preferred_batch_size: [1024, 8192]\n",
+    "  max_queue_delay_microseconds: 100\n",
+    "}\n",
+    "```\n",
+    "Store the above config at `/data/model_repository/fil/` directory as config.pbtxt as follows:\n",
```

Comment:
Let's generate this file for them (either in the notebook or externally) so they don't have to go through the hassle of figuring that out inside the Docker container or thinking about how the bind-mount works.

Reply:
Ah, I see. It's happening in the following cell. See my above comment about signposting.